lib/formatter/csv/annotation_for_csv.rb

```diff
@@ -125,7 +131,8 @@ def task_label(task_info)
         translate(task_info["question"] || task_info["instruction"])
       end
 
-      def tool_label(task_info, tool_index)
+      def tool_label(task_info, tool)
+        tool_index = tool["tool"]
         have_tool_lookup_info = !!(task_info["tools"] && tool_index)
         known_tool = have_tool_lookup_info && task_info["tools"][tool_index]
         translate(known_tool["label"]) if known_tool
```

Comment:
That case should not happen as we have a correct tool label (it matches a lookup value in the task tools) and those tool labels should have been setup correctly when creating the tasks object for the workflow. 

If the tool label does not match up to a know index / tool then it'll return nil and no tool label will be recorded in the annotation. Basically if it's a weird annotation and the tool label doesn't match then we don't try and make a guess. I could return something like 'unknown_tool' but i don't think that helps the researcher (no traceability) and it just bloats the export.